python/ray/worker.py

```diff
@@ -674,6 +688,50 @@ def export_remote_function(self, function_id, function_name, function,
             })
         self.redis_client.rpush("Exports", key)
 
+    def register_class_on_all_workers(self, function):
```

Comment:
It's okay to implement `RegisterType` using the same mechanism as `FunctionToRun`.

But since the purpose of this function is to register serializer for a type, I think it'd make more sense to use the type, the serializer, etc as the parameters, instead of using a function.

Also, looks like that some code is copy-pasted from `run_function_on_all_workers`, let's also unify the 2 functions?